src/js/services/feeService.js

```diff
@@ -1,8 +1,15 @@
 'use strict';
 
-angular.module('copayApp.services').factory('feeService', function($log, profileService, configService) {
+angular.module('copayApp.services').factory('feeService', function($log, profileService, configService, gettextCatalog) {
   var root = {};
 
+  // Constant fee options to translate
+  var feeOpts = [
+    gettextCatalog.getString('priority'),
```

Comment:
The source returns theses words in lower case. So, to avoid modify in the html, the best way is to translate as it comes.